test/libsolidity/syntaxTests/literalSuffixes/parameters/integer_out_of_range.sol

```diff
@@ -16,10 +16,10 @@ contract C {
     }
 }
 // ----
-// TypeError 8838: (364-370): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (380-386): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (396-405): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (415-424): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (434-516): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 5503: (536-619): This fractional number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function.
-// TypeError 8838: (536-619): The type of the literal cannot be converted to the parameter of the suffix function.
+// TypeError 8838: (364-370): The number is out of range of type int8 accepted by the suffix function.
+// TypeError 8838: (380-386): The number is out of range of type uint8 accepted by the suffix function.
+// TypeError 8838: (396-405): The number is out of range of type int16 accepted by the suffix function.
+// TypeError 8838: (415-424): The number is out of range of type uint16 accepted by the suffix function.
+// TypeError 8838: (434-516): The number is out of range of type int256 accepted by the suffix function.
+// TypeError 5503: (536-619): This number cannot be decomposed into a mantissa and decimal exponent that fit the range of parameters of any possible suffix function. The mantissa is out of range of the largest supported integer type.
+// TypeError 8838: (536-619): The number is out of range of type uint256 accepted by the suffix function.
```

Comment:
Again, the first message should not appear here because the suffix takes only one argument.

Reply:
fixed in [6d144c7](https://github.com/ethereum/solidity/pull/14015/commits/6d144c75d6b19003e38e286eb7b037e8b92ac76b).